Scroller/EditableWrapper.module.less

```
@@ -0,0 +1,32 @@
+// EditableWrapper.module.less
+//
+
+.wrapper {
+	display: flex;
+	flex-direction: row;
+	min-width: fit-content;
+    padding-top: 60px;
+}
+
+.centered {
+	justify-content: center;
+}
+
+.selected {
+	transition: transform 360ms;
+	transform: translateY(-48px);
+	will-change: transform;
+	z-index: 1;
+}
+
+.selectedTransform {
+	transform: translate(var(--selected-item-offset), -48px);
+}
```

Let's unify with selected

```suggestion
.selected {
	transition: transform 360ms;
	transform: translate(var(--selected-item-offset), -48px);
	transform: translateY(-48px);
	will-change: transform;
	z-index: 1;
	transform: translate(var(--selected-item-offset), -48px);
}
```